client/src/amplify-ui/cli-feature-flag/feature-flags.json

```diff
@@ -84,6 +84,27 @@
             "defaultExistingProject": true
           }
         ]
+      },
+      "secondaryKeyAsGSI": {
+        "description": "Changes the behaviour of @key to remove the creation of an LSI.",
+        "type": "Feature",
+        "valueType": "Boolean",
+        "versionAdded": "4.41.x",
+        "deprecationDate": "May 1st 2021",
+        "values": [
+          {
+            "value": "true",
+            "description": "Will default to creating a GSI when using @key",
+            "defaultNewProject": true,
+            "defaultExistingProject": false
+          },
+          {
+            "value": "false",
+            "description": "Will use old behavior to create a LSI when the using a primary field.",
```

Comment:
Will use old behavior of creating an LSI when an @key directive uses a primary key field.